# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/Navigator.cpp
# Commit: 0c5428075ca4
# Full Hash: 0c5428075ca4aa6b5270350247d43217fa5d0feb
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-07 08:31:23
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 9e7fe3dbea01 -r 0c5428075ca4 dom/base/Navigator.cpp
--- a/dom/base/Navigator.cpp	Fri Jun 06 14:38:54 2025 +0000
+++ b/dom/base/Navigator.cpp	Fri Jun 06 14:38:54 2025 +0000
@@ -2269,7 +2269,7 @@
 
 NavigatorLogin* Navigator::Login() {
   if (!mLogin) {
-    mLogin = new NavigatorLogin(GetWindow()->AsGlobal());
+    mLogin = new NavigatorLogin(GetWindow());
   }
   return mLogin;
 }